app/scripts/views/marketing_snippet_ios.js

```diff
@@ -80,6 +85,12 @@ define(function (require, exports, module) {
       };
     },
 
+    submit: function () {
+      this.logViewEvent('sms');
```

Comment:
@shane-tomlinson @seanmonstar Other views already log all submits, I think this should too without adding this new event. Should add a test that the submit event is logged.